crypto/keyring/keyring.go

```diff
@@ -907,9 +914,11 @@ func (ks keystore) migrate(version uint32, i keyring.Item, migrated bool) (bool,
 
 		legacyInfo, err := unmarshalInfo(item.Data)
 		if err != nil {
-			unmarshalErr := fmt.Errorf("unmarshalInfo(item.Data), err - %s", err)
+			unmarshalErr := fmt.Errorf("unable to unmarshal item.Data, err: %w", err)
 			fmt.Println(unmarshalErr)
-			continue
```

Comment:
According to @robert-zaremba migration [algorithm](https://github.com/cosmos/cosmos-sdk/pull/9222#discussion_r624683839) i have to continue here. I am not sure if it is proper behavior here. I would suggest to output an error. Any feedback?

Reply:
we continue because this can be a rubbish key - we don't want to break the process if we encounter a rubbish in the keyring.